src/ast/query.rs

```diff
@@ -278,6 +283,13 @@ impl fmt::Display for TableFactor {
                 }
                 Ok(())
             }
+            TableFactor::TableFunction { expr, alias } => {
+                write!(f, "TABLE({})", expr)?;
```

Comment:
Wouldn't it be more consistent to add a space after `TABLE`?

Reply:
This is currently formatted like a function call and follows the formatting used in snowflake's documentation. What do you think it should it be consistent with?

Reply:
I don't care about removing this whitespace or not .. 
whats looks better for you - it's fine for me ..

Reply:
I think the function call inside the `TABLE` should have no space, but after it it is just a parenthesized expression? So I would say, to be consistent with the docs (which also has a space after table) and other constructs a space makes sense here.

Reply:
Ah I see the snowflake documentation now. That too makes sense if you see it like a function call... Maybe in docs for the enum, it could have the space removed too?

Reply:
I removed the space from the enum definition.